pkg/obfuscate/sql.go

```diff
@@ -57,7 +57,7 @@ func (f *metadataFinderFilter) Filter(token, lastToken TokenKind, buffer []byte)
 			// SELECT ... FROM [tableName]
 			// DELETE FROM [tableName]
 			// ... JOIN [tableName]
-			if r, _ := utf8.DecodeRune(buffer); !unicode.IsLetter(r) {
+			if r, _ := utf8.DecodeRune(buffer); !unicode.IsLetter(r) && !strings.ContainsRune("\"'`", r) {
 				// first character in buffer is not a letter; we might have a nested
```

Comment:
Can you update the comment to reflect the newly added condition? This code is quite intricate and having helpful comments is of great value.